src/gui/transferlistfilterswidget.cpp

```diff
@@ -420,8 +423,11 @@ void TrackerFiltersList::handleFavicoDownload(const QString& url, const QString&
 void TrackerFiltersList::handleFavicoFailure(const QString& url, const QString& error)
 {
     Q_UNUSED(error)
-    if (url.endsWith(".ico", Qt::CaseInsensitive))
+    if (url.endsWith(".ico", Qt::CaseInsensitive)) {
         downloadFavicon(url.left(url.size() - 4) + ".png");
+    } else {
+        downloadFavicon("https://www.google.com/s2/favicons?domain=" + getHost(url));
```

Comment:
I just though you will declare a constant with that URL and it is natural to re-use it, that's all.